### PR TITLE
[TS] LPS-134368

### PR DESCRIPTION
**Pull request by rolandpakai**
Hi Team,

could you please check my fix.
Because of not handling the context path loading css fails.
Adding context path fix the issue.

https://issues.liferay.com/browse/LPS-134368

### CODE REVIEW

File: modules/apps/layout/layout-taglib/src/main/java/com/liferay/layout/taglib/internal/display/context/RenderLayoutStructureDisplayContext.java [hunk 1]
```diff
@@ -748,7 +748,7 @@ public String getStyle(StyledLayoutStructureItem styledLayoutStructureItem)
 			styleSB.append("background-position: 50% 50%; background-repeat: ");
 			styleSB.append("no-repeat; background-size: cover; ");
 			styleSB.append("background-image: url(");
-			styleSB.append(backgroundImage);
+			styleSB.append(PortalUtil.getPathContext() + backgroundImage);
```
Comment (diegonvs) [inline modules/apps/layout/layout-taglib/src/main/java/com/liferay/layout/taglib/internal/display/context/RenderLayoutStructureDisplayContext.java:751]: Hey @rolandpakai, what do you think about moving this getPathContext call [to the return of `_getBackgroundImage ` ](https://github.com/liferay-frontend/liferay-portal/blob/7b93f45172e3f03683783a12c3b9d8254dd1177e/modules/apps/layout/layout-taglib/src/main/java/com/liferay/layout/taglib/internal/display/context/RenderLayoutStructureDisplayContext.java#L1061)function?

Comment (rolandpakai) [reply modules/apps/layout/layout-taglib/src/main/java/com/liferay/layout/taglib/internal/display/context/RenderLayoutStructureDisplayContext.java:751]: Hi @diegonvs No problem, we can move it there

**Comment by diegonvs** [comment]
Re-running relevant suite to just make sure the CI failure is not flaky

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by rolandpakai** [comment]
ci:test:relevant

**Comment by rolandpakai** [comment]
Hi @diegonvs 

could you please push this forward?

Thanks, Roland

**Comment by diegonvs** [comment]
Hey @rolandpakai, I don't have access to push things to your fork. I'll create another PR :)

**Comment by diegonvs** [comment]
Closing in favor of #1234 (interesting sequence, reminds me [Feist - 1234](https://www.youtube.com/watch?v=ABYnqp-bxvg))

**Comment by diegonvs** [comment]
Closing in favor of #1234 (interesting sequence, reminds me [Feist - 1234](https://www.youtube.com/watch?v=ABYnqp-bxvg))

**Pull request closed without merging**